tests/vmi_configuration_test.go

```diff
@@ -590,7 +584,51 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 				By("Checking the guest ACPI SLIC table matches the one provided")
 				Expect(console.SafeExpectBatch(vmi, []expect.Batcher{
 					&expect.BSnd{S: "xxd -p -c 40 /sys/firmware/acpi/tables/SLIC\n"},
-					&expect.BExp{R: console.RetValue(hexData)},
+					&expect.BExp{R: console.RetValue(hex.EncodeToString(slicTable))},
+				}, 3)).To(Succeed())
+			})
+
+			It("Should configure guest ACPI MSDM with Secret file", func() {
```

Comment:
Consider doing a table test for SLIC and MSDM or just merge them.

Reply:
@xpivarc at first I thought about merging them because they are similar, but in the end they are different features. In the unit tests, we have them together to be sure we are not doing anything wrong wrt libvirt APIs.

The table test can be done, just need to add more logic to the test params as each ACPI table has different headers. If you don't mind, I'll consider it a follow-up just to avoid holding this before the code freeze